src/js/utils/browser.js

```diff
@@ -4,7 +4,7 @@
 import document from 'global/document';
 import window from 'global/window';
 
-const USER_AGENT = window.navigator.userAgent;
+export const USER_AGENT = window.navigator.userAgent;
```

Comment:
Nope, [the `browser` object](https://github.com/videojs/video.js/blob/master/src/js/video.js#L219) includes all the exported constants here.